env/facets-dev-2019.sh

```diff
@@ -0,0 +1,2 @@
+export CBIOPORTAL_URL="http://dashi-dev.cbio.mskcc.org:8080/ascn-portal"
```

Comment:
Just a note, do not forget to remove this script before merging to master branch. I don't think we should even commit anything includes dashi servers, but I guess it's ok for the convenience sake.

Reply:
@averyniceday make sure to remove this

Reply:
I guess this is a requirement to have the tests to pass otherwise the current production does not have the data? But anyway, we need to remove before merging to master.